src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,108 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+
+    @BeforeEach
+    public void setUp() {
+        user = new User();
+        user.setAge(18);
+        user.setLogin("123456");
+        user.setPassword("123456");
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_ok() {
+        registrationService.register(user);
+        User stored = Storage.people.get(0);
+        assertNotNull(stored, "User should be stored in Storage");
+    }
```

Comment:
The method name 'register_ok' does not follow the naming convention from the checklist. It should be something like 'register_validUser_ok'.